credential_issuer/health.keys.json

```diff
@@ -0,0 +1,3 @@
+{
+    "timestamp_srv": "http://apiroom.net:3312"
```

Comment:
ahaha this is not reliable at all ;p do a query on the server itself, a localhost... so you check that is tautologically doing a self-assesment plz!